CHANGELOG.md

```diff
@@ -6,9 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+
+- **CUMULUS-2428**
+  - Changed `/granules/bulk` to use `queueUrl` property instead of a `queueName` property for setting the queue to use for scheduling bulk granule workflows
```

Comment:
I hate flagging this as a breaking change since technically the `queueName` property wasn't actually being used at all by the code, but technically this is a breaking change to the payload expected/used by our API

Reply:
As I thought about it more, the fact that `queueName` wasn't being used due to a bug is kind of a detail. It's still a breaking change to the API either way, so as a user if you did have code making requests with `queueName`, you would have to change it to `queueUrl`.

Reply:
Yeah, totally agree. If it was exposed this should be marked as a breaking change.